medusa/tv/episode.py

```diff
@@ -269,7 +271,10 @@ def download_subtitles(self, lang=None):
                         ep=(episode_num(self.season, self.episode) or
                             episode_num(self.season, self.episode, numbering='absolute')))
 
-            notifiers.notify_subtitle_download(self.pretty_name(), subtitle_list)
+            try:
+                notifiers.notify_subtitle_download(self.pretty_name(), subtitle_list)
```

Comment:
Honestly I don't know how a notification can ever throw an exception. That smells like bad design.
As a notification is only needed, when you have something a notify on. Maybe using some kind of callback, can fix this.